map/discovery/discovery_manager.cpp

```diff
@@ -27,28 +27,16 @@ Manager::Manager(Index const & index, search::CityFinder & cityFinder, APIs cons
 }
 
 // static
-search::SearchParams Manager::GetSearchParams(Manager::Params const & params, ItemType const type)
+search::DiscoverySearchParams Manager::GetSearchParams(Manager::Params const & params, ItemType const type)
 {
-  search::SearchParams p;
+  search::DiscoverySearchParams p;
   p.m_query = GetQuery(type);
-  p.m_inputLocale = "en";
   p.m_viewport = params.m_viewport;
-  p.m_position = params.m_viewportCenter;
-  p.m_maxNumResults = params.m_itemsCount;
-  p.m_mode = search::Mode::Viewport;
-  return p;
-}
+  p.m_viewportCenter = params.m_viewportCenter;
```

Comment:
BTW, what's the reason to pass viewportCenter and viewport as individual params? It should not be hard to calculate center of a rectangle, right?

Reply:
Aha, it seems that m_viewportCenter should be renamed to m_position?